lib/matplotlib/tests/test_contour.py

```diff
@@ -693,3 +693,10 @@ def test_contour_remove():
     assert ax.get_children() != orig_children
     cs.remove()
     assert ax.get_children() == orig_children
+
+
+def test_contour_no_args():
+    fig, ax = plt.subplots()
+    data = [[0, 1], [1, 0]]
+    with pytest.raises(TypeError):
```

Comment:
```suggestion
    with pytest.raises(TypeError, match="contour\(\) takes 1 to 4":
```
(if that is the correct way to escape regexes...)

Reply:
Please squash the commits if possible. Otherwise we'll do that upon merge.

Reply:
Done and done!